src/Platforms/SqlitePlatform.php

```diff
@@ -34,11 +34,11 @@
 /**
  * The SqlitePlatform class describes the specifics and dialects of the SQLite
  * database platform.
- *
- * @todo   Rename: SQLitePlatform
```

Comment:
Please revert.

Reply:
I will - as a separate PR, ok?

Reply:
> as a separate PR, ok?

No, why should we introduce a change just to revert it in another PR?